packages/sanity/src/core/form/inputs/files/ImageToolInput/imagetool/makeDragAware.tsx

```diff
@@ -28,34 +25,17 @@ export function makeDragAware(Component: 'canvas') {
     isDragging = false
 
     componentDidMount() {
-      if (supportsTouch) {
-        document.body.addEventListener('touchmove', this.handleTouchMove, {passive: false})
-        document.body.addEventListener('touchend', this.handleDragEnd)
-        document.body.addEventListener('touchcancel', this.handleDragCancel)
-      } else {
-        document.body.addEventListener('mousemove', this.handleDrag)
-        document.body.addEventListener('mouseup', this.handleDragEnd)
-        document.body.addEventListener('mouseleave', this.handleDragCancel)
-      }
+      document.body.addEventListener('pointermove', this.handleDrag)
+      document.body.addEventListener('pointerup', this.handleDragEnd)
+      document.body.addEventListener('pointerleave', this.handleDragCancel)
+      document.body.addEventListener('pointercancel', this.handleDragCancel)
     }
 
     componentWillUnmount() {
-      if (supportsTouch) {
-        document.body.removeEventListener('touchmove', this.handleTouchMove)
-        document.body.removeEventListener('touchend', this.handleDragEnd)
-        document.body.removeEventListener('touchcancel', this.handleDragCancel)
-      } else {
-        document.body.removeEventListener('mousemove', this.handleDrag)
-        document.body.removeEventListener('mouseup', this.handleDragEnd)
-        document.body.removeEventListener('mouseleave', this.handleDragCancel)
-      }
-    }
-
-    handleTouchMove = (event: TouchEvent | MouseEvent) => {
-      // Disables mobile scroll by touch
-      if (this.isDragging) {
-        event.preventDefault()
-      }
```

Comment:
This was removed in favour of using the [touch-action](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action) CSS property to prevent some (but not all) touch events from being handled by the browser.

Alternatively, we. could set this to `touch-action: none` but that would prevent pinch-zooming, which can be helpful (especially if you need help targeting those draggable nodes)

Reply:
TIL about the touch-action, thanks for sharing! Agree we should preserve pinch-zooming.